content/sensu-go/6.2/operations/control-access/create-limited-service-accounts.md

```diff
@@ -0,0 +1,332 @@
+---
+title: "Create limited service accounts with role-based access control (RBAC)"
+linkTitle: "Create Limited Service Accounts"
+guide_title: "Create limited service accounts with role-based access control (RBAC)"
+type: "guide"
+description: "Role-based access control (RBAC) allows you to create limited service accounts so that applications can access and interact with Sensu resources. Read this guide to create limited service accounts with Sensu RBAC."
+weight: 65
+version: "6.2"
+product: "Sensu Go"
+platformContent: false
+menu: 
+  sensu-go-6.2:
+    parent: control-access
+---
+
+In some cases, you may want to allow an application or service to interact with Sensu resources.
+Sensu's [role-based access control (RBAC)][1] allows you to exercise fine-grained control over access to and interaction with resources.
+You can use RBAC to set up limited service accounts, which are accounts that represent applications or services rather than individual human users.
+
+For example, you might develop a service that displays a high-level view of your webserver statuses based on an aggregate check.
+The service itself only needs an API key and permission to read the results of status checks executed on your webservers so it can route the check results to the status display.
+No human user needs to log into the service, and the service does not need edit or delete permissions.
+You can create a limited service account to provide the necessary access.
+
+Limited service accounts are also useful for performing automated processes.
+This guide explains how to create a limited service account to use with the [Sensu EC2 Handler][3] integration to automatically remove AWS EC2 instances that are not in a pending or running state.
+
+By default, Sensu includes a `default` namespace and an `admin` user with full permissions to create, modify, and delete resources within Sensu, including the RBAC resources required to configure a limited service account.
+This guide requires a running Sensu backend and a sensuctl instance configured to connect to the backend as the [`admin` user][2].
+
+## Create a limited service account
+
+A limited service account requires:
+
+- A [user][7] assigned to a [group][8].
+- A [role][4] with get, list, and delete permissions for resources within the `default` [namespace][9].
+- A [role binding][5] that ties the role to the user's group.
```

Comment:
Since we are not required to have a group, this should be "ties the role to the user."